themes/cypress/layout/media.swig

```diff
@@ -3,143 +3,141 @@
     <div id="content-inner">
```

Comment:
are the changes in this file related to the PR or can they be reverted?

Reply:
This is necessary for the experimental styling that you're seeing on the page.